src/components/ui/DeviceMotionView.tsx

```diff
@@ -0,0 +1,72 @@
+import { Text, View } from 'react-native';
+import React, { useState, useEffect } from 'react';
+import { DeviceMotion } from 'expo-sensors';
+import { useBackgroundTasks } from "@src/components/providers/BackgroundTasksProvider";
+import * as Haptics from "expo-haptics";
+
+export default function DeviceMotionView() {
+    const [{beta, gamma, alpha }, setRotationData] = useState({beta: 0, gamma: 0, alpha: 0 });
```

Comment:
if alpha is not being used, remove its declaration

Reply:
This is updated.